### PR TITLE
[EB-4011] update for timezone function

### PR DESCRIPTION
**Pull request by julianideamotive**
Changes for getDateTime in timezone, as discussed on Slack, we will use moment object there because we use moment in many places in app

### CODE REVIEW

**Comment by pncsantos** [comment]
@julianideamotive , can we retain getUserDateTimefromUTC function and deprecate it later on after the front end is updated.

**Comment by pncsantos** [comment]
@julianideamotive : can we deprecate getUserDateTimefromUTC function later on once we update all the timezone function to avoid breaking changes in the frontend.

**Comment by julianideamotive** [comment]
@pncsantos yes, it is removed from app, app on branch EB-4011 should use new function instead. It isn't used anywhere in sdk.

**Comment by julianideamotive** [comment]
@pncsantos I've left the old function to prevent breaking changes

File: src/data/utility/__test__/util.test.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {
   convertObjectIntoKeyValueArray,
   customError,
   mergeArraysWithObjects,
-  getUserDateTimefromUTC,
```
Comment (maartenhemmes) [inline src/data/utility/__test__/util.test.js]: @julianideamotive shouldn't we still also test the other function?

Comment (julianideamotive) [reply src/data/utility/__test__/util.test.js]: @maartenhemmes tests added

**Pull request merged**